docs/content/en/docs/tasks/cluster/etcd-backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Etcd Backup and Restore"
+linkTitle: "Etcd Backup and Restore"
+weight: 11
+date: 2021-11-04
+---
+
+This page contains steps for backing up a cluster by taking an etcd snapshot, and restoring the cluster from a snapshot. These steps are for an EKS Anywhere cluster provisioned using the external etcd topology (selected by default) and Ubuntu OVAs.
+
+### Use case
+
+EKS-Anywhere clusters use etcd as the backing store. Taking a snapshot of etcd backs up the entire cluster data. This can later be used to restore a cluster back to an earlier state if required. Etcd backups can be taken prior to cluster upgrade, so if the upgrade doesn't go as planned you can restore from the backup.
+
+
+### Backup
+
+Etcd offers built-in snapshot mechanism. You can take a snapshot using the `etcdctl snapshot save` command by following the steps given below. 
+
+1. Login to any one of the etcd VMs
+```
+ssh -i $PRIV_KEY ec2-user@$ETCD_VM_IP
+```
+2. Run the etcdctl command to take a snapshot with the following steps
+```
+sudo su
+source /etc/etcd/etcdctl.env
+etcdctl snapshot save snapshot.db
+chown ec2-user .
```

Comment:
Just curious about why you are changing ownership of the current directory to the ec2-user. Do you mean to change the ownership of the snapshot.db file? Because I don't think that would work.